modules/BUILD.bazel

```diff
@@ -6,7 +6,7 @@ package_group(
         # All other modules.
         "//modules/...",
         # Project generation.
-        "@rules_xcodeproj_generated//...",
+        "@rules_xcodeproj//:generated",
```

Comment:
Needs to be in `includes`

Reply:
This is what I get for trying to just use Codespaces to do it all without ever having it locally.